config/clients/js/template/configuration.mustache

```diff
@@ -58,18 +65,28 @@ export class Configuration {
    */
   private static sdkVersion = "{{packageVersion}}";
 
+  /**
+   * provide the full api URL
```

Comment:
Can you append the example here?

`provide the full api URL (e.g. `https://api.{{sampleApiDomain}}`)`

Reply:
Fixed in 9ad45dc73c895d6f47060852bc42994ff3ce025e